dogpile/cache/region.py

```diff
@@ -879,6 +882,7 @@ def get_value():
                     value.metadata["ct"]
                 )
             ):
+                log.info('Missing cache for key: %s', orig_key)
```

Comment:
If I'm viewing this log, wouldn't I want to know the difference between a. NO VALUE b. value was present but being cleared by previous dogpile version (value_version) c. value was present but is hard invalidated?